test/functional/apps/visualize/_data_table_notimeindex_filters.js

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
Curious, if there is a specific reason this file is `.js`, not `.ts`?

Reply:
Copied an existing test, due to short time frame for release.
Will look into it. Thanks!